pages/_app.tsx

```diff
@@ -27,6 +27,7 @@ import ProjectPropsProvider, {
 } from '../src/features/common/Layout/ProjectPropsContext';
 import UserPropsProvider from '../src/features/common/Layout/UserPropsContext';
 import PlayButton from '../src/features/common/LandingVideo/PlayButton';
+import ErrorHandlingProvider from '../src/features/common/Layout/ErrorHandlingContext';
```

Comment:
Have you verified that the error is still also handled by Sentry?

Reply:
Yes, this is just a way to inform the user. will not affect sentry functionality.

Reply:
This is a base for project-wide error management. We can later have custom error logs and reports from this. and it won't affect sentry.